work3.py

```diff
@@ -6,4 +6,5 @@ def get_middle_three_chars(str1):
     print("Middle three chars are:", res)
 
 get_middle_three_chars("JhonDiePeta")
-get_middle_three_chars("JaSonAy")
\ No newline at end of file
+get_middle_three_chars("JaSonAy")
+#comment
```

Comment:
was this change necessary - i dont see any significant change just a comment added

Reply:
LOL